ta/pkcs11/src/processing.c

```diff
@@ -104,6 +104,19 @@ void release_active_processing(struct pkcs11_session *session)
 	if (!session->processing)
 		return;
 
+	switch (session->processing->mecha_type) {
+	case PKCS11_CKM_RSA_PKCS_PSS:
+	case PKCS11_CKM_SHA1_RSA_PKCS_PSS:
+	case PKCS11_CKM_SHA256_RSA_PKCS_PSS:
+	case PKCS11_CKM_SHA384_RSA_PKCS_PSS:
+	case PKCS11_CKM_SHA512_RSA_PKCS_PSS:
+	case PKCS11_CKM_SHA224_RSA_PKCS_PSS:
+		tee_release_rsa_pss_operation(session->processing);
```

Comment:
I think `tee_release_rsa_pss_operation()` and this switch/case are not needed.
It is called only from here (from `release_active_processing()`) which already frees `processing->extra_ctx` , at line 131.

Reply:
OK.

Will do the same then for OAEP.

Shall we have a need for specific release then we can add special case for that in future.